src/nft/pages/collection/index.tsx

```diff
@@ -34,8 +42,30 @@ const Collection = () => {
           <CollectionStats stats={collectionStats} isMobile={isMobile} />
         </Row>
       )}
-      <Row alignItems="flex-start" position="relative" paddingLeft="32" paddingRight="32">
-        <AnimatedBox width="full">
+      <Row alignItems="flex-start" position="relative" paddingX="48">
+        <Box position="sticky" top="72" width="0">
+          {isFiltersExpanded && <Filters />}
+        </Box>
+
+        {/* @ts-ignore */}
```

Comment:
fyi this is needed because there is a bug with `tsserver`

Reply:
wat

Reply:
https://github.com/microsoft/TypeScript/issues/34933

Reply:
i guess just add a comment linking to this